File: a20241107a(未整合的校正腳本)/plotCalbPoint.m
function  plotCalbPoint(xy)
num=length(xy);
x=xy(:,1);y=-xy(:,2);
plot(x,y,'+');
axis([0 640 -480 0])

for n=1:num
    sn=num2str(n);
    text(x(n),y(n),sn,'FontSize',6)
end

File: a20241107a(未整合的校正腳本)/plotCalbPoint.py
import matplotlib.pyplot as plt
import numpy as np

def plotCalbPoint(xy):
    num = len(xy)
    x = xy[:, 0]
    y = -xy[:, 1]
    plt.plot(x, y, '+')
    plt.axis([0, 640, -480, 0])

    for n in range(num):
        sn = str(n + 1)  # MATLAB 是 1 索引
        plt.text(x[n], y[n], sn, fontsize=6)

    plt.show()